docs/getting-started-scalardl-auditor.md

```diff
@@ -0,0 +1,545 @@
+
+In this guide, we will create the following environment in your local by using Docker and minikube.  
+
+```
+On the Docker Network (named minikube)
+
+                   +---------------------------------------------+    +-----------+    +-----------+
+  +-----------+    | +------------------+  +-------------------+ |    |           |    |           |
+  |           |    | | Scalar DL Ledger |  | Scalar DL Auditor | |    | Cassandra |    | Cassandra |
+  |  Client   |    | +------------------+  +-------------------+ |    | Container |    | Container |
+  | Container |    |                                             |    | (Ledger)  |    | (Auditor) |
+  |           |    |            Kubernetes (minikube)            |    |           |    |           |
+  +-----+-----+    +----------------------+----------------------+    +-----+-----+    +-----+-----+
+        |                                 |                                 |                |
+*-------+---------------------------------+---------------------------------+----------------+-------*
+             ------->                                           ------->
+               gRPC                                          Cassandra Driver
+```
+
+## Step 1. Create the Scalar DL Ledger environment
+
+First, you need to create a Scalar DL Ledger environment with Auditor configuration.  
+Please refer to the `Step 1` ~ `Step 7` of [Getting Started with Helm Charts (Scalar DL Ledger)](./getting-started-scalardl-ledger.md) to create a Ledger environment.  
+To enable Auditor configuration in the Scalar DL Ledger, set `true` to `ledger.scalarLedgerConfiguration.ledgerAuditorEnabled` in `Step 7` of the above guide.  
+
+After this step, we will create (add) a Scalar DL Auditor environment into the above Scalar DL Ledger environment.  
+Also, after this step, we call `Scalar DL Ledger` as `Ledger` and `Scalar DL Auditor` as `Auditor`.  
+
+## Step 2. Start Cassandra container for Auditor
+
+In this guide, we use Apache Cassandra as backend storage of the Auditor, it will be started on the same network of Auditor (Pod on minikube) to enable proper communication.
+
+1. Start Cassandra container for Auditor on the Docker Network `minikube`.
+   ```console
+   $ docker run --name cassandra-auditor --network minikube -d cassandra:3.11
+   ```
+   * Note: 
+       * The Docker Network `minikube` was created by the `minikube start --driver=docker` command that we ran in Step 1.
+       * You need to specify the Cassandra version (tag) that the Scalar DB that is used in Scalar DL supports.
+           * You can see the version of Scalar DB that is used in each Scalar DL in the [build.gradle](https://github.com/scalar-labs/scalar/blob/master/build.gradle) file (see the value of `scalarDbVersion`).
+           * Also, you can see the Cassandra version that is supported by Scalar DB in [this document](https://github.com/scalar-labs/scalardb/blob/master/docs/scalardb-supported-databases.md).
```

Comment:
Ditto

Reply:
I fixed in 8538ab98e531f0a02391952adbc58601cac0ae0b.